ddl/db_change_test.go

```diff
@@ -743,7 +743,7 @@ func (s *testStateChangeSuite) testControlParallelExecSQL(c *C, sql1, sql2 strin
 
 	_, err = s.se.Execute(context.Background(), "drop database if exists t_part")
 	c.Assert(err, IsNil)
-	_, err = s.se.Execute(context.Background(), `create table t_part (a int key)
+	s.se.Execute(context.Background(), `create table t_part (a int key)
```

Comment:
I think we should keep the original version and check `err` is nil. Or maybe a better change is to Call `MustExec()`?

Reply:
It seems that it will be called by some other tests, and the `err` here is not always `nil`?